openmetadata-airflow-apis/openmetadata_managed_apis/api/routes/last_dag_logs.py

```diff
@@ -0,0 +1,49 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+Health endpoint. Globally accessible
+"""
+import logging
+import traceback
+
+from airflow.api_connexion import security
+from airflow.security import permissions
+from airflow.www.app import csrf
+from flask import Response, request
+from openmetadata_managed_apis.api.app import blueprint
+from openmetadata_managed_apis.api.response import ApiResponse
+from openmetadata_managed_apis.api.utils import clean_dag_id, get_request_arg
+from openmetadata_managed_apis.operations.last_dag_logs import last_dag_logs
+
+
+@blueprint.route("/last_dag_logs", methods=["GET"])
+@csrf.exempt
+@security.requires_access([(permissions.ACTION_CAN_READ, permissions.RESOURCE_DAG)])
+def last_logs() -> Response:
+    """
+    Retrieve all logs from the task instances of a last DAG run
+    """
+    raw_dag_id: str = get_request_arg(request, "dag_id")
+
+    if not raw_dag_id:
+        ApiResponse.bad_request("Missing dag_id parameter in the request")
```

Comment:
I think this check could be part of the get_request_arg method if the request argument is mandatory, wdyt?. In addition, it looks like we have a few methods to get the `dag_id` in the utils class.

Reply:
we have one method for picking it from POST jsons and another as a query arg